### DISCUSSION TITLE
EditDialog.tsx file not compiling

### DISCUSSION THREAD

**Discussion by rhemasystems** [serenity-is/Serenity#7299] [Help]
I have this strange issue with one of my `.tsx` files, breakpoint works in every other .tsx file except this particular `editdialog.tsx` file, even if i delete the file, the application still compiles and runs smoothly except that any new code changes to that file are not compiled.
I realised the issue is not with dubugging because `editor.tsx` hits breakpoint without issues.

I have deleted `bin`,`obj `hidden `.vs` folders but still 
I even deleted it from the `Imports\ClientTypes` folder but it wasnt replaced after rebuild
Any assistance will greatly be appreciated @VictorTomaili, @volkanceylan 

`Editor.tsx`
![image](https://github.com/user-attachments/assets/81f6625e-9f13-4447-9509-71553118fcb2)

`EditDialog.tsx`

![image](https://github.com/user-attachments/assets/281e7d98-dbd6-43bb-9e6a-c7a98aed5e21)

**Comment by VictorTomaili** [comment]
Don't mention users in comment to get help.

This is because of your project is not built successfully or your typescript has other issue which cause typescript build error. Also check csproj file for included excluded typescript files. And don't use exclude files from project. Delete them instead, you should already use source control software (like git) to find the file in  the future.

Additional note: When you have an issue, try to reproduce it in the new testing project from template and describe how to reproduce. You have no details about your environment and it is not possible to know what happened. If you cannot reproduce it, this means the issue is related your project. Try to figure out what is the difference between your project and the testing project. 

I close this, you can open it when you have more details.
